pkgs/top-level/aliases.nix

```diff
@@ -661,6 +669,7 @@ mapAliases ({
   nylas-mail-bin = throw "nylas-mail-bin was deprecated on 2019-09-11: abandoned by upstream";
   oauth2_proxy = oauth2-proxy; # added 2021-04-18
   opencascade_oce = opencascade; # added 2018-04-25
+  obconf = throw "obconf has been removed"; # added 2021-01-16
```

Comment:
Make that 2022 :)

Reply:
Oh no, muscle memory :see_no_evil: And I consistently did it wrong everywhere